analysis/topEFT/topeft.py

```diff
@@ -362,9 +365,10 @@ def process(self, events):
         bmask_atleast2med = (nbtagsm>=2) # Used for 3l SR
 
         # Charge masks
-        charge2l_p = ak.fill_none(((l0.charge+l1.charge)>0),False)
-        charge2l_m = ak.fill_none(((l0.charge+l1.charge)<0),False)
+        charge2l_p = ak.fill_none(((l0.charge)>0),False)
+        charge2l_m = ak.fill_none(((l0.charge)<0),False)
         charge2l_0 = ak.fill_none(((l0.charge+l1.charge)==0),False)
+        charge2l_pm = ak.fill_none(((l0.charge+l1.charge)!=0),False)
```

Comment:
We should think of a better name. To me `pm` is plus/minus, meaning opposite sign.

Reply:
thanks! I just pushed the change